src/utils/error.ts

```diff
@@ -0,0 +1,27 @@
+import { ParserError } from "../record/import/repositories/parsers/error";
+
+export abstract class CliKintoneError extends Error {
```

Comment:
@ueokande 
I removed overridden `toString()` methods.
Because some inherited Errors need to parse `this.cause`, i added the protected method `_toStringCause()`